github/orgs_teams.go

```diff
@@ -114,10 +114,37 @@ func (s *OrganizationsService) GetTeam(ctx context.Context, team int) (*Team, *R
 	return t, resp, nil
 }
 
+// NewTeam represents a team to be created or modified.
+type NewTeam struct {
+	Name         string   `json:"name"` // Name of the team (Required.)
+	Description  *string  `json:"description,omitempty"`
+	Maintainers  []string `json:"maintainers,omitempty"`
+	RepoNames    []string `json:"repo_names,omitempty"`
+	ParentTeamID *string  `json:"parent_team_id,omitempty"`
+
+	// Permission is deprecated when creating or editing a team in an org
+	// using the new GitHub permission model. It no longer identifies the
+	// permission a team has on its repos, but only specifies the default
+	// permission a repo is initially added with. Avoid confusion by
+	// specifying a permission value when calling AddTeamRepo.
```

Comment:
@e-beach Are you familiar with the [convention Go uses for deprecated symbols](https://github.com/golang/blog/commit/257114af91a0defb1fc2c16c7f4ae2429b8a4e0f)?

I think it would be nice to use here.

Reply:
That is copied from `Team` so I'll fix it there as well:
https://github.com/google/go-github/blob/511f540f1887d30b88cee4a2fcd1f2922754acf4/github/orgs_teams.go#L23-L28